x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -267,27 +312,98 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
-              onChange={() => setFormState({ createIndexPattern: !createIndexPattern })}
-            />
-          </EuiFormRow>
+          {jobType === JOB_TYPES.REGRESSION && (
+            <Fragment>
+              <EuiFormRow
+                label={i18n.translate(
+                  'xpack.ml.dataframe.analytics.create.dependentVariableLabel',
+                  {
+                    defaultMessage: 'Dependent variable',
+                  }
+                )}
+                error={
+                  dependentVariableFetchFail === true && [
+                    <Fragment>
+                      {i18n.translate(
+                        'xpack.ml.dataframe.analytics.create.dependentVariableOptionsFetchError',
+                        {
+                          defaultMessage: 'There was a problem fetching fields. Please try again.',
+                        }
+                      )}
+                    </Fragment>,
+                  ]
+                }
+              >
+                <EuiComboBox
+                  aria-label={i18n.translate(
+                    'xpack.ml.dataframe.analytics.create.dependentVariableInputAriaLabel',
+                    {
+                      defaultMessage: 'Enter field to be used as dependent variable.',
+                    }
+                  )}
+                  placeholder={i18n.translate(
+                    'xpack.ml.dataframe.analytics.create.dependentVariablePlaceholder',
+                    {
+                      defaultMessage: 'dependent variable',
+                    }
+                  )}
+                  isDisabled={isJobCreated}
+                  isLoading={loadingDepFieldOptions}
+                  singleSelection={true}
+                  options={dependentVariableOptions}
+                  selectedOptions={dependentVariable ? [{ label: dependentVariable }] : []}
+                  onChange={selectedOptions =>
+                    setFormState({ dependentVariable: selectedOptions[0].label || '' })
+                  }
+                  isClearable={false}
+                  isInvalid={dependentVariable === ''}
+                />
+              </EuiFormRow>
+              <EuiFormRow
+                label={i18n.translate('xpack.ml.dataframe.analytics.create.trainingPercentLabel', {
+                  defaultMessage: 'Training percent',
+                })}
+              >
+                <EuiRange
+                  min={0}
+                  max={100}
+                  step={1}
+                  showLabels
+                  showRange
+                  showValue
+                  value={trainingPercent}
+                  // @ts-ignore Property 'value' does not exist on type 'EventTarget' | (EventTarget & HTMLInputElement)
+                  onChange={e => setFormState({ trainingPercent: e.target.value })}
+                />
+              </EuiFormRow>
+            </Fragment>
+          )}
+          {jobType === JOB_TYPES.OUTLIER_DETECTION && (
```

Comment:
Not sure about specs or limitations, but should index pattern creation also be available for regression type jobs?

Reply:
Yep - done in 6de9d0d162db3f4225bd32b347bdd502f4d51719